USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -0,0 +1,250 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using USFMToolsSharp.Models.Markers;
+using NPOI.XWPF.UserModel;
+using System.IO;
+
+namespace USFMToolsSharp.Renderers.Docx
+{
+    public class DocxRenderer
+    {
+        public List<string> UnrenderableTags;
+        public Dictionary<string,List<Marker>> FootnoteTextTags;
+        
+        private XWPFDocument newDoc;
+
+        private bool isSingleSpaced = true;
+        private bool hasOneColumn = true;
+        private bool isL2RDirection = true;
+        private bool isTextJustified = false;
+        private bool separateChapters = true;
+
+        public string FrontMatterDocx { get; set; }
```

Comment:
Doesn't look like this is used. Inserting the front matter is going to be interesting 😄